### PR TITLE
NCD - HbA1c lab

### PR DESCRIPTION
**Pull request by anvmn**
#546

### CODE REVIEW

**Comment by anvmn** [comment]
Peek 2022-12-11 15-34

**Comment by anvmn** [comment]
@mariano-dagostino please review

File: client/src/elm/Translate.elm [hunk 1]
```diff
@@ -5138,6 +5142,11 @@ translationSet trans =
             , kinyarwanda = Just "Kujya ahabanza"
             }
 
+        GotResultsPreviouslyQuestion ->
+            { english = "Has patiend perviously performed HBA1C test and got results"
```
Comment (mariano-dagostino) [inline client/src/elm/Translate.elm]: @anvmn `Has patient previously performed HBA1C test and got results`

**Review by mariano-dagostino — APPROVED** [review]
@anvmn Looks good, one minor string to fix.

**Pull request merged**